src/main/resources/templates/fragments/footer.html

```diff
@@ -5,7 +5,7 @@
     <title>Title</title>
 </head>
 <body>
-<footer class="footer fixed-bottom mt-auto py-3" th:fragment="footer">
+<footer class="footer mt-auto py-3" th:fragment="footer">
```

Comment:
Removed the fixed-bottom class from the footer to align the page more comprehensively.